### PR TITLE
Feature/st 00031 dsp fields

### PR DESCRIPTION
**Pull request by rthomp-prosci**
1. Creates the following custom fields:
* Deal_Support_Process__c.Amount__c
* Deal_Support_Process__c.Price__c
* Deal_Support_Process__c.Quantity__c
* Deal_Support_Process__c.Opportunity_Product__c

2. Grants Global Growth users access to the new custom fields, via Global Growth Base Permission Set

### CODE REVIEW

File: force-app/main/default/objects/Contact/Contact.object-meta.xml
Comment (jnelson-prosci) [inline force-app/main/default/objects/Contact/Contact.object-meta.xml:1]: Let's not commit the contact object metadata.

File: force-app/main/default/objects/Deal_Support_Process__c/Deal_Support_Process__c.object-meta.xml
Comment (jnelson-prosci) [inline force-app/main/default/objects/Deal_Support_Process__c/Deal_Support_Process__c.object-meta.xml:1]: Is this necessary for this PR too?

File: force-app/main/default/objects/Lead/recordTypes/Enterprise.recordType-meta.xml
Comment (jnelson-prosci) [inline force-app/main/default/objects/Lead/recordTypes/Enterprise.recordType-meta.xml:1]: these record type commits look wrong to me. Try retrieving the object with the record types in order to get the appropriate field level updates for the recordtype metadata file.

**Comment by rthomp-prosci** [comment]
Thanks, Jimmy! All three comments relate to the same commit (802dbfb), which is not crucial to this PR. That commit is now reverted.

File: force-app/main/default/flexipages/Deal_Support_Process_Page_Single_Related_Lists.flexipage-meta.xml [hunk 1]
```diff
@@ -117,29 +117,6 @@
...
-                    </criteria>
-                    <criteria>
-                        <leftValue>{!Record.Status__c}</leftValue>
-                        <operator>EQUAL</operator>
-                        <rightValue>Closed - Lost</rightValue>
-                    </criteria>
-                </visibilityRule>
-            </fieldInstance>
-        </itemInstances>
         <itemInstances>
```
Comment (jnelson-prosci) [inline force-app/main/default/flexipages/Deal_Support_Process_Page_Single_Related_Lists.flexipage-meta.xml:120]: @wchuprosci -- Did you just add this today? I'm guessing we don't want to remove it

Comment (wchuprosci) [reply force-app/main/default/flexipages/Deal_Support_Process_Page_Single_Related_Lists.flexipage-meta.xml:120]: Yup these were part of my changes to add the naming logic for training events, please keep this in the layout

**Review by jnelson-prosci — APPROVED** [review]

File: force-app/main/default/flexipages/Deal_Support_Process_Page_Single_Related_Lists.flexipage-meta.xml (hunk 1 above)
Comment (rthomp-prosci) [reply force-app/main/default/flexipages/Deal_Support_Process_Page_Single_Related_Lists.flexipage-meta.xml]: This change is now restored, per commit 08bdd03. Thank you both!

**Pull request merged**